src/state/lottery/index.ts

```diff
@@ -85,6 +85,14 @@ export const fetchUserLotteries = createAsyncThunk<
   return userLotteries
 })
 
+export const fetchAdditionalUserLotteries = createAsyncThunk<
```

Comment:
Minor thing, but we can just call this `fetchUserLotteries`

Reply:
There is already a `fetchUserLotteries` - the data fetch and the reducer behave a bit differently